MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -1162,6 +1162,7 @@
 	_metalFeatures.mslVersion = SPIRV_CROSS_NAMESPACE::CompilerMSL::Options::make_msl_version(maj, min);
 
 	switch (_metalFeatures.mslVersionEnum) {
+		default:
```

Comment:
Present a warning about missing case for `MTLLanguageVersion2_3`

Reply:
The `default:` case is deliberately left off here for exactly this reason...to get warnings when new MSL versions are added, so that we remember to include them.

With that in mind, I've created a new [`xcode12`](https://github.com/KhronosGroup/MoltenVK/tree/xcode12) branch to this repo, where we can add all sorts of new functionality using the API's supported by Xcode 12, including `MTLLanguageVersion2_3`.

Later this year, once Xcode 12 and the OS versions it supports are released, I'll merge that branch with master for general use.

Can you add `MTLLanguageVersion2_3` appropriately to the top of the switch statement in this function, and submit a new pull request targeting the `xcode12` branch of this repo, please?

If you want to be thorough, also update `MVKPhysicalDevice::initMetalFeatures()` to include the setting of `_metalFeatures.mslVersionEnum` to `MTLLanguageVersion2_3`, following the pattern used to set it to `MTLLanguageVersion2_2` using `mvkOSVersionIsAtLeast()`. You'll need to set it in 3 places, one for each OS. However, this action is not strictly necessary for this PR, since your goal is just to get Xcode 12 working, and we can set `MTLLanguageVersion2_3 ` in a future PR.

Reply:
fixed